### ISSUE TITLE
use data-attributes instead of `href: javascript()` on anchor elements

### ISSUE THREAD

**Issue by stefanprobst** [acdh-oeaw/vicav-app-api#16]
currently, the text endpoints return an html string, which contains anchor elements like this:

```html
<a
  class="aVicText"
  href="javascript:getDBSnippet(&quot;text:vicavDictionaryEncoding/DICTIONARY_ENCODING&quot;, this)"
  data-target-type="Text"
  data-text-id="vicavDictionaryEncoding/DICTIONARY_ENCODING"
  data-label=""
>
  detailed of our lexical data
</a>
```

all information needed by a client should be passed via data attributes, and the href should not be a `javascript()` call.

**Comment by stefanprobst** [comment]
also, please clarify how to use the `text-id` data attribute to resolve to a resource.

```bash
https://vicav.acdh-ch-dev.oeaw.ac.at/vicav/text?id=vicavDictionaryEncoding/DICTIONARY_ENCODING
```

does not seem to work.

**Comment by simar0at** [comment]
This was an implementation error. The data was not split correctly.  Fixed in [vicav-app/f44e06b](https://github.com/acdh-oeaw/vicav-app/commit/f44e06bb21c2718dd2f05bbc5fdd552fcbf1f93c)
